pandas/core/internals/managers.py

```diff
@@ -352,6 +353,12 @@ def reduce(self, func, *args, **kwargs):
 
         return res
 
+    def double_apply(self, other: "BlockManager", array_op) -> "BlockManager":
```

Comment:
I find the `double_apply` not a very illuminating name. Maybe just `operate_blockwise` (or `apply_with_blockmanager`)? I am not sure the "apply" necessarily needs to be in the name, since this is much more limited in functionality as the other `apply` method.

Reply:
> Maybe just operate_blockwise

sure